em-pg-client-helper.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
 
 	s.add_runtime_dependency "em-pg-client",     "~> 0.3"
 	s.add_runtime_dependency "git-version-bump", "~> 0.10"
-	s.add_runtime_dependency "sequel",           ">= 3.31.0", "< 5"
+	s.add_runtime_dependency "sequel",           ">= 3.31.0", "< 5.38.0"
```

Comment:
I would probably be less particular than this; as soon as 5.39 comes out, we'll need to update this gem again.

Instead, something like `< 6` would work for everything in the 5.x series.